ee/codegen/src/__test__/nodes/__snapshots__/tool-calling-node.test.ts.snap

```diff
@@ -71,6 +71,18 @@ class ToolCallingNode(BaseToolCallingNode):
 "
 `;
 
+exports[`ToolCallingNode > deployment workflow > should generate latest release tag if release_tag is null 1`] = `
+"from vellum.workflows.nodes.experimental.tool_calling_node import (
+    ToolCallingNode as BaseToolCallingNode,
+)
+from vellum.workflows.types.definition import DeploymentDefinition
+
+
+class ToolCallingNode(BaseToolCallingNode):
+    functions = [DeploymentDefinition(deployment="deployment_1", release_tag="LATEST")]
```

Comment:
[nit] I'd slightly prefer codegenning `DeploymentDefinition(deployment="deployment_1")`, and updating our SDK logic to know that no `release_tag` defaults to `"LATEST"` when left unspecified